lametro/people.py

```diff
@@ -83,7 +83,8 @@ def scrape(self):
 
         for body in self.bodies():
             if body['BodyTypeId'] == body_types['Committee']:
-                o = Organization(body['BodyName'],
+                org_name = body['BodyName'].strip()
```

Comment:
Yes, we want to remove leading and trailing whitespace (i.e., if an indexed facet has whitespace in the front or back, the Haystack search returns zero results, since the inner bowels of Haystack clean both sides of the facet name). For reference, here's the Metro issue this PR addresses: https://github.com/datamade/la-metro-councilmatic/issues/150 (cf. comments from two days ago).